src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,9 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+
+  width: 1024px;
+
+  margin: 0 auto;
```

Comment:
Again margin in a block (.cards). Should be moved to an element. I see there are other the same kinda mistake in other blocks as well.

Reply:
I removed this margin, didn't need it apparently to center :D